architecture-building-systems/CityEnergyAnalyst
# -remove custom-roof-coverage parameter

This solves #3179.
A boolean parameter that was reduntant was removed.

How to review:
1. check the code changes
2. Open on the dashboard (energy potantials -> photovoltaic panels / photovoltaic-thermal potantials)
3. Test simulation with different values for roof-coverage and confirm there are changes.
